src/components/preview/preview.test.js

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import Preview from "./preview.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import previewComp from "../../../cypress/locators/preview/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const testData1 = [256, 275, 300];
+
+const PreviewComponent = ({ ...props }) => {
+  return <Preview loading {...props} />;
+};
+
+context("Tests for Preview component", () => {
+  describe("should check Preview component properties", () => {
+    it.each(testData1)(
+      "should check height as %spx for Preview component",
+      (height) => {
+        CypressMountWithProviders(<PreviewComponent height={height} />);
+        previewComp().should("have.css", "height", `${height}px`);
+      }
+    );
+
+    it.each(testData1)(
+      "should check width as %spx for Preview component",
+      (width) => {
+        CypressMountWithProviders(<PreviewComponent width={width} />);
+        previewComp().should("have.css", "width", `${width}px`);
+      }
+    );
+
+    it.each(testData)(
+      "should check children as %s for Preview component",
+      (children) => {
+        CypressMountWithProviders(<Preview>{children}</Preview>);
+        cy.get("#__cy_root").should("have.text", children);
+      }
+    );
+
+    it.each([true, false])(
+      "should check when loading is %s for Preview component",
+      (bool) => {
+        CypressMountWithProviders(<PreviewComponent loading={bool} />);
+        if (bool === true) {
+          previewComp().should("be.visible");
+        } else {
+          previewComp().should("not.exist");
+        }
+      }
+    );
+
+    it.each([5, 6, 8, 10])(
+      "should check %s loading lines for Preview component",
+      (line) => {
+        CypressMountWithProviders(<PreviewComponent lines={line} />);
+        previewComp()
```

Comment:
those tests have same assertion for each run? How could it be possible to have same amount of lines for 5,6,8 and 10 scenarios ?